cli/dts/lib.deno.unstable.d.ts

```diff
@@ -333,8 +331,20 @@ declare namespace Deno {
     | "usize"
     | "isize"
     | "f32"
-    | "f64"
-    | "pointer";
+    | "f64";
+
+  /** All possible types for interfacing with foreign functions */
+  export type NativeType =
+    | NativeNumberType
+    | "void"
+    | "pointer"
+    | NativeStructType;
+
+  export interface NativeStructType<
```

Comment:
Absolutely 100% totally unimportant nitpick, but maybe put all the constituent parts above the actual `NativeType` type?

Reply:
Makes it easier to specifically handle `NativeStructType` than repeating `{ struct: ... }` everywhere